package.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "clue",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "git@github.com:itstheandre/lab-javascript-clue.git",
+  "author": "André de Albuquerque <itstheandre@gmail.com>",
+  "license": "MIT",
```

Since we want Ironhack labs to be used by Ironhack students and staff exclusively, in the future we'll be creating a custom license. Until then, we should license our labs as "UNLICENSED (https://github.com/npm/npm/issues/8918#issuecomment-148550718).

```suggestion
  "license": "UNLICENSED",
```